src/GraphQLParser/AST/Values/GraphQLBooleanValue.cs

```diff
@@ -6,15 +6,25 @@ namespace GraphQLParser.AST;
 /// AST node for <see cref="ASTNodeKind.BooleanValue"/>.
 /// </summary>
 [DebuggerDisplay("GraphQLBooleanValue: {Value}")]
-public class GraphQLBooleanValue : GraphQLValue
+public class GraphQLBooleanValue : GraphQLValue, IHasValueNode
```

Comment:
Now that we do not allow changing a boolean, we could have `GraphQLBooleanValue` abstract, and have separate derived classes for true and false variations.  I don't really care either way but it could possibly save some tiny bit of memory.

Perhaps separate PR even if we do want to do that.

Reply:
I did not do this because there will be combinations with descendant internal classes . In any case, if desired, this can be done later.

Reply:
#195